apps/test-suite/TestUtils.js

```diff
@@ -70,8 +70,11 @@ export function getTestModules() {
   if (ExponentTest && !ExponentTest.isInCI) {
     // Invalid placementId in CI (all tests fail)
     modules.push(require('./tests/FBNativeAd'));
-    // Requires interaction (sign in popup)
-    modules.push(require('./tests/GoogleSignIn'));
+
+    if (Constants.isDevice && Platform.OS !== 'android') {
```

Comment:
Doesn't this still run on DeviceFarm?

Reply:
I don't think so. Looks like Linking is the error now